x-pack/plugins/security_solution/public/resolver/store/selectors.ts

```diff
@@ -155,26 +157,6 @@ export const relatedEventsByCategory = composeSelectors(
   dataSelectors.relatedEventsByCategory
 );
 
-/**
- * Entity ids to booleans for waiting status
- * @deprecated
- */
-export const relatedEventsReady = composeSelectors(
```

Comment:
:grey_question: This is replaced at-large by the thing you did where the whole panel spins in a waiting state right?

Reply:
the panel will load until it's finds data. error states aren't done. this code is a stopgap to keep things working until kq gets his PR in. He should be doing something with the data abstraction layer changes we discussed.